### PR TITLE
Isolated component refactor

### PR DESCRIPTION
**Pull request by DAquilinaCinchy**
Refactor of the dynamic field components in an isolated context so that they can be merged into the 5.4.0 release candidate individually.

### CODE REVIEW

File: src/app/dynamic-forms/service/form-helper/form-helper.service.ts [hunk 1]
```diff
@@ -126,7 +126,7 @@ export class FormHelperService {
         formFields[i].childFormLinkId,
         formFields[i].doNotWrap,
         columnMetadata?.displayFormat,
-        columnMetadata?.$type == 'Calculated'
+        columnMetadata?.$type == 'Calculate'
```
Comment (alvin-h) [inline src/app/dynamic-forms/service/form-helper/form-helper.service.ts:129]: Is this correct? It's not "_Calculated_"?

File: src/app/dynamic-forms/service/print/print.service.ts [hunk 2]
```diff
@@ -384,21 +392,6 @@ export class PrintService {
     } else if (value && currentField && currentField.cinchyColumn.dataFormatType === 'LinkUrl') {
       return `<a href="${value}" target="_blank">Open</a>`;
     }
-    else if(this.isAnchor(value)){
```
Comment (alvin-h) [inline src/app/dynamic-forms/service/print/print.service.ts:387]: I think we need this, this was code @cinchy-yogeshjagtap added to address a bug with rendering anchor elements... could it be that branch isn't up to date?

**Comment by alvin-h** [comment]
The cleanup makes sense to me, I'm good with everything minus the two other comments. But question is, why the preference for double quotes vs single quotes?

File: src/app/dynamic-forms/service/print/print.service.ts (hunk 2 above)
Comment (DAquilinaCinchy) [reply src/app/dynamic-forms/service/print/print.service.ts]: This is an issue with the branch interactions. I've corrected this.

**Comment by DAquilinaCinchy** [comment]
> 

Mostly it goes back to my C/C++ days (where single quotes indicated a character and double quotes indicated a string), but more practically, it's a better default. The average string is more likely to have an apostrophe than quotation marks, so you're less likely to need to escape characters this way.

**Review by alvin-h — APPROVED** [review]

**Pull request closed without merging**